src/top-bar/context-select/context-select.js

```diff
@@ -0,0 +1,80 @@
+import {
+    IconChevronDown24,
+    IconChevronUp24,
+    colors,
+    Popover,
+    Tooltip,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useRef } from 'react'
+import classes from './context-select.module.css'
+
+const ContextSelect = ({
+    children,
+    label,
+    value,
+    onClose,
+    onOpen,
+    disabled,
+    open,
+    tooltipContent,
+}) => {
+    const buttonRef = useRef()
+    const Icon = open ? IconChevronUp24 : IconChevronDown24
+    const button = (
+        <button
+            ref={buttonRef}
+            className={classes.button}
+            onClick={onOpen}
+            disabled={disabled}
+        >
+            <span className={classes.prefix}>{label}</span>
+            {!disabled && <span className={classes.value}>{value}</span>}
+            <Icon color={disabled ? colors.grey600 : undefined} />
+        </button>
+    )
+
+    return (
+        <>
+            {disabled ? (
+                <Tooltip content={tooltipContent} placement="bottom">
```

Comment:
I don't get the connection between `disabled` and `tooltipContent`. Is the content actually sth like an explanation for why the button is disabled?

Reply:
Ok, from reading how the org unit select works, I guess my assumption is correct. Maybe `requiredValuesLabel` or `requirementsLabel` makes more sense? Hides the implementation detail that it's going to be in a tooltip, creates a connection between `disabled` and the tooltip's content and just makes sense when using the component.